### PR TITLE
Implement E2E Test cases and GH actions

### PR DESCRIPTION
**Pull request by alvitazwar**

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# e2e test for Wp-menu-custom-fields
+
+name: CI for Wp-menu-custom-fields
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [automation/playwright]
```
Comment (pooja-muchandikar) [inline .github/workflows/e2e.yml:9]: @alvitazwar Update the branch name to develop. As we are merging this PR in develop and any PR raised against develop we are going to run tests on that branch.

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  pull_request:
+    branches: [automation/playwright]
```
Comment (pooja-muchandikar) [inline .github/workflows/e2e.yml:11]: @alvitazwar same here, update branch name

**Review by pooja-muchandikar — COMMENTED** [review]
@alvitazwar Left few feedbacks..

Also Remove [artifact folder](https://github.com/rtCamp/wp-menu-custom-fields/pull/94/files#diff-f172e5b94fcff9a73723f36ea6df99f514827d35c4bf35802b4daaf48758c9b2). It's not needed.

File: .github/workflows/e2e.yml (hunk 2 above)
Comment (alvitazwar) [reply .github/workflows/e2e.yml]: Done

**Review by pooja-muchandikar — APPROVED** [review]

**Pull request merged**